pkgs/applications/misc/whalebird/default.nix

```
@@ -83,8 +83,8 @@ stdenv.mkDerivation rec {
     description = "Electron based Mastodon, Pleroma and Misskey client for Windows, Mac and Linux";
     homepage = "https://whalebird.social";
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
-    license = licenses.mit;
-    maintainers = with maintainers; [ wolfangaukang colinsane ];
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Only;
```
`gpl3` is [deprecated](https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206). and looking upstream, they seem to have licensed it under "GPL 3.0" without any mention of "or any later version".